### DISCUSSION TITLE
note/video-format-and-play/

### DISCUSSION THREAD

**Discussion by giscus** [gitguanqi/gitguanqi.github.io#12] [Announcements]
# note/video-format-and-play/

导语：之前做过一个视频播放的小项目，其中用到了ffmpeg的工具，就好奇着想了解一下常见的视频格式都有哪些，现在就总结一下常见的视频格式以及播放的方法。
目录
视频格式
播放软件
视频格式
MPGE格式
MPEG(Moving Picture Experts Group)，全称是运动图像专家组格式，采用有损压缩方法，减少图像中的冗 ...

https://guanqi.xyz/note/video-format-and-play/

**Comment by gitguanqi** [comment]
### 视频软件总结

仔细看了看
写的还行啊！

**Reply by gitguanqi** [reply]
真棒

**Comment by gitguanqi** [comment]
挺全面的，软件也挺多的

**Comment by gitguanqi** [comment]
加油啊，继续写好的文章
